homework/v.paslavskiy/Hometask1.1.py

```diff
@@ -0,0 +1,101 @@
+name = None
+answer1 = None
+answer2 = None
+answer3 = None
+answer4 = None
+answer5 = None
+if name == None:
+    name = input('What is your name?:  ' )
+    age = int(input("Hello {}, how old are you?:  ".format(name)))
+    if age > 0 and age <= 7:
+        while answer1 == 'a' or 'b' or 'c':
+            answer1 = input("""What are you doing here, little baby?
+                                a) I'm looking for my mommy!
+                                b) I'm looking for some cookies
+                                c) I'm lost
+                            """)
+            if answer1 != 'a' or 'b' or 'c':
```

Comment:
this expression will always be True unless answer 1 is not 'a'

Reply:
For such condition, you could use
while answer1 not in ['a','b','c'] 
or use AND operator.

But since you use 'break' for flow control, you could just use 'while True' and some 'else' at the end of the switch.